folium/features.py

```diff
@@ -652,6 +682,70 @@ def __init__(self, html=None, icon_size=None, icon_anchor=None,
         self.className = class_name
 
 
+class Tooltip:
+    """
+Creates a Tooltip object for adding to features to display text as a property a Map by executing a javascript function
+when hovering the cursor over each feature.
+
+Parameters
+----------
```

Comment:
Please fix the indentation of the docstrings.

Reply:
@ocefpaf think I fixed most of the issues. Thanks for your patience - still new to deploying standard code for public use, but learning a lot with this.